WorkingInR/0_Preamble.Rmd

```diff
@@ -0,0 +1,74 @@
+---
+title: "Preamble"
+output:
+  bookdown::html_document2:
+  highlight: tango
+toc: true
+toc_float: true
+css: ../css/style-chapters.css
+---
+  
+``` {r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE)
+```
+
+
+# Preamble
+
+## About this website
+
+These pages are the result of a collaborative effort between bioCEED, Centre for Excellence in Biology Education, the Department of Biological Sciences at the University of Bergen, and the Department of Biology and the Department of Mathematical Sciences at the Norwegian University of Science and Technology.
+
+## Who is it for?
+
+
+
+## Prerequisites
+
++ functional computer and access to the internet (sais in a humoristic)
++ stats 101 ?
+
+
+## Structure of the webpages
+
+### Before we start
+
+At the top of each page, you will find the `Before we start` box which tells you what you must already know or be familiar with before you read further. 
+
+### Body
+
+The body of the page introduces the topic(s) along with relevant examples.
+
+### Interactive apps 
+
+R is a language; thus it is **important to practice writing** to learn it better and faster. 
+To do so, we have included a set of interactive apps which you may use to try the code. 
+Figure \@ref(fig:app0) shows how such an interactive app looks like.
+ 
+```{r app0, echo = FALSE, eval = TRUE, fig.cap="_Screenshot of an interactive app._", out.width="100%"}
+knitr::include_graphics("Pics/app0.png")
+```
+
+Sometimes, the app will be pre-filled with the full code ready to run. 
```

Comment:
Maybe also say that sometimes there are tips, and you can see the solution by clicking on the solution button.

Reply:
I've added a few minor changes to the first part of the text, and the following to the end:

Some of these applications are set up as more or less complex exercises. 
Two additional buttons may be available: `Hint` and `Solution`. 
`Hint` may provide you with fragments of the code or tips that will point you in the right direction; `Solution` reveals the answer.